react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+
+export function StopWatch() {
+  const [seconds, setSeconds] = useState(0);
+  const [play, setPlay] = useState(false);
+  const [pauseID, setPauseID] = useState(null);
+
+  function startTimer() {
+    if (!play) {
+      const count = setInterval(() => {
+        setSeconds((seconds) => seconds + 1);
+      }, 1000);
+      setPlay(true);
+      setPauseID(count);
+    }
+    clearInterval(pauseID);
```

Comment:
Should this be inside an `else`? Otherwise, I think you could call `clearInterval` with an invalid ID. Though that seems to work, MDN doesn't tell us what happens if you do that, so it's best to be safe.